engine/runtime-compiler/src/main/scala/org/enso/compiler/PackageRepository.scala

```diff
@@ -95,12 +95,12 @@ trait PackageRepository {
   def isNamespaceRegistered(namespace: String): Boolean
 
   /** Returns a package directory corresponding to the requested library */
-  def getPackageForLibrary(lib: LibraryName): Option[Package[TruffleFile]]
+  def getPackageForLibrary(lib: LibraryName): Option[Package[File]]
 
   /** Returns a package directory corresponding to the requested library */
   def getPackageForLibraryJava(
     libraryName: LibraryName
-  ): java.util.Optional[Package[TruffleFile]] =
+  ): java.util.Optional[Package[File]] =
```

Comment:
I'd integrate `fansi` removal first and only then worked on Truffle dependencies. As I am not sure if `Package[File]` is the best approach...

Reply:
Will be addressed in another issue - #8888